### DISCUSSION TITLE
Official PCB, do you sell it?

### DISCUSSION THREAD

**Discussion by raybantracing** [sblantipodi/glow_worm_luciferin#41] [Q&A] [answered]
Do you sell a pre-built official PCB for Luciferin?

Thanks

**Comment by sblantipodi** [comment, marked as answer]
Hi,
no there is no pre-built PCB but it's pretty easy to do built it.

There is a wiki page here:
https://github.com/sblantipodi/firefly_luciferin/wiki/Ready-to-print-PCB

And a YouTube video that shows how to solder the components here:
https://youtu.be/J4TE75ME86w

Bye
